src/cloudai/workloads/megatron_run/megatron_run.py

```
@@ -81,8 +81,16 @@ def no_dashed_args(self):
     @property
     def cmd_args(self) -> dict[str, Union[str, list[str]]]:
         args = self.model_dump(exclude_none=True, exclude={"docker_image_url", "run_script"})
-        args = {f"--{k.replace('_', '-')}": v for k, v in args.items()}
-        return args
+        result: dict[str, Union[str, list[str]]] = {}
+        for k, v in args.items():
+            flag = f"--{k.replace('_', '-')}"
+            if v == "false":
+                continue
+            elif v in ("true", ""):
```

Ruff (0.15.21)

[warning] Unnecessary `elif` after `continue` statement

Remove unnecessary `elif`

(RET507)